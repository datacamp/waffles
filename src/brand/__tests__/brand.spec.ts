import * as brands from '../index';

describe('Brand', () => {
  it('exports every Brand component', () => {
    expect(brands).toMatchSnapshot();
  });
});
