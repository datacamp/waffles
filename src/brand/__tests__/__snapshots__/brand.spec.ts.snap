// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand exports every Brand component 1`] = `
Object {
  "DataCampLogo": [Function],
  "DataCampLogoTagline": [Function],
  "DataCampLogomark": [Function],
  "DataCampLogomarkTagline": [Function],
}
`;
